charts/node-local-dns/values.yaml

```diff
@@ -73,3 +73,19 @@ config:
   # Could be retrieved with `kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}`
   dnsServer: ""
   clusterDomain: cluster.local
+
+serviceMonitor:
+  # -- If true, a ServiceMonitor CRD is created for a prometheus operator.
+  # https://github.com/coreos/prometheus-operator
+  #
+  enabled: false
+  # -- HTTP scheme to use for scraping.
+  scheme: http
+  # -- Labels to add to ServiceMonitor.
+  labels: {}
+  # -- Interval at which metrics should be scraped.
+  interval: 15s
+  # -- Timeout after which the scrape is ended.
+  scrapeTimeout: 15s
```

Comment:
We should probably let these to "" (ie the server decides) by default ?

Reply:
I don't agree with that, a default value seems more convenient for the user.

Reply:
I said that while comparing to these charts, I found many of them not setting theses values (so I did not previously) : 
* https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L350
* https://github.com/grafana/helm-charts/blob/main/charts/loki/values.yaml#L221

But indeed some are also setting their values for interval (https://github.com/bitnami/charts/blob/master/bitnami/minio/values.yaml#L888), even though it is still a very low default value (it's usually 30s, or the prometheus default 1m). 
I did not see timeout set by default though, but I don't know why (I think the prometheus default is 10s and the rule is to not be above the interval).

Reply:
Feel free to contribute !